src/Pact/Native/Db.hs

```diff
@@ -354,12 +364,16 @@ guardTable i t = evalError' i $ "Internal error: guardTable called with non-tabl
 
 enforceBlessedHashes :: FunApp -> ModuleName -> Hash -> Eval e ()
 enforceBlessedHashes i mn h = do
-  mmRs <- fmap fst . HM.lookup mn <$> view (eeRefStore.rsModules)
+  mmRs <- fmap _mdModule . HM.lookup mn <$> view (eeRefStore . rsModules) -- fmap fst . HM.lookup mn <$> view (eeRefStore.rsModules)
```

Comment:
commented code?

Reply:
I wanted to discuss whether this was equivalent at one point. I believe we answered that. I'll remove it